templates/projects/tasks_browse.webpack.ejs

```diff
@@ -2,9 +2,11 @@
 {% set active_page = "projects" %}
 {% set active_project  = "all" %}
 {% set active_link = "tasks" %}
-{% set section = _('Browse tasks') %}
 {% import "projects/_helpers.html" as project_helper %}
 
+{% set is_task_list = request.args['view'] == 'tasklist' %}
+{% set section = _('Task List') if is_task_list else 'Browse Tasks' %}
```

Comment:
How about putting those two lines before line 5 so all set statements group together? Also ```'Browse Tasks'``` could be ```_('Browse Tasks')```, for consistency. I guess "_" is a special function.

Reply:
Sounds good - thanks Xi!